src/components/app.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable func-style */
```

Comment:
Fix this linter error. Disabling the linter rules changes the linter config which is used by all Microverse students.